### PR TITLE
lxd stop console support

### PR DESCRIPTION
**Pull request by Fl4v**

### CODE REVIEW

File: lxc/action.go [hunk 1]
```diff
@@ -245,6 +249,16 @@ func (c *cmdAction) doAction(action string, conf *config.Config, nameArg string)
 		return err
 	}
 
+	if action == "stop" {
+		// Handle console attach
+		if c.flagConsole != "" {
```
Comment (stgraber) [inline lxc/action.go:254]: Those two if statements can be turned into one.

Comment (Fl4v) [reply lxc/action.go]: Done

**Comment by stgraber** [comment]
Looks good. Can you rebase everything you have into a single commit:
 - lxc/action: Allow --console with stop action

Make sure to include `Closes #8546` in the commit message.

Then add another commit on top by first running `make i18n` and then running:
 - git commit -a -m "i18n: Update translation templates"

That should get us the two commits we expect for such a change.

Thanks!

**Comment by Fl4v** [comment]
Sure thing!

**Comment by Fl4v** [comment]
@stgraber All done. I hope it's what you expect.

**Comment by stgraber** [comment]
Ah, one more thing, your commits must be signed-off.
You can possibly do that using `git rebase --sign-off HEAD^^` or by directly editing them and adding:
```
Signed-off-by: Firstname Lastname <email>
``

To both of them.

**Comment by Fl4v** [comment]
Easy enough. I will keep that in mind for next time. Thank you for your help on this feature!

**Comment by stgraber** [comment]
Looks like you're missing a line break in there, Github looks confused by the commit messages.
Should be something like:

```
lxc/action: Allow --console with stop action.

Closes #8546.

Signed-off-by: Flavius Stroe <stroe.flavius@gmail.com>
```

**Comment by Fl4v** [comment]
Like this?

**Comment by stgraber** [comment]
Yep, looking good!

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
All tests are failing with:
```
/lxc-ci/build/tmp.OgTMk456zx/go/bin/lxc stop baz --force --verbose
Error: --console can't be used while forcing instance shutdown
```

File: lxc/action.go [hunk 2]
```diff
@@ -202,6 +202,10 @@ func (c *cmdAction) doAction(action string, conf *config.Config, nameArg string)
 		state = true
 	}
 
+	if action == "stop" && c.flagForce {
```
Comment (stgraber) [inline lxc/action.go:205]: Oh, that's missing a `&& c.flagConsole` :)

Comment (Fl4v) [reply lxc/action.go:205]: Oh, that's my bad. I will fix in the morning if that's ok.

Comment (stgraber) [reply lxc/action.go:205]: Yep, no worries.

Comment (Fl4v) [reply lxc/action.go:205]: @stgraber Should I re-run `make i18n` after I have changed the if statement?

Comment (stgraber) [reply lxc/action.go:205]: No, this doesn't change the translated string so no need to re-run.

Comment (Fl4v) [reply lxc/action.go]: Hi @stgraber, I think I have made a mistake and the commit to fix the if statement is in this commit: `i18n: Update translation templates. ` Will this be a problem?

Comment (Fl4v) [reply lxc/action.go]: I think I have kind of answered my own question, I will try fix and it.

Comment (Fl4v) [reply lxc/action.go]: All done. Apologies for the mess.

**Comment by stgraber** [comment]
jenkins: test this please

**Pull request merged**